products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,232 @@
+ORDER BY DeathCount DESC
+```
+
+2) Now that we have the location with the most deaths, let's list the episodes with the most deaths
+as well.
+
+```opencypher
+MATCH (d:Death)-[:InEpisode]->(e:Episode)
+RETURN e.name, COUNT(d) AS KillCount
+ORDER BY KillCount DESC
+```
+
+3) List the number of kills per season. I think we all know the highest count one. 
+
+```opencypher
+MATCH (d:Death)-[:InSeason]->(s:Season) 
+RETURN s.number AS SeasonNumber, COUNT(d) as DeathNumber
+ORDER BY SeasonNumber ASC
+```
+
+4) While we know which season was by far the worst, can you guess the best one based on user ratings?
+Let's list the seasons by IMDB ratings. The problem we get with using the `AVG()` function is that it
+gives us too many decimals, therefore a useful solution is given in this example using `ROUND()`.
+
+```opencypher
+MATCH (e:Episode)-[:InSeason]->(s:Season) 
+RETURN s.number AS SeasonName, ROUND(100 * AVG(e.imdb_rating))/100 AS Rating
+ORDER BY Rating DESC
+```
+
+5) There are many methods characters died with such as with sword or Dragonfire, but let's list victims
+of unique methods.
+
+```opencypher
+MATCH (c:Character)-[k:Killed]->(c2:Character)
+WITH k.method AS Method, COUNT(k.method) AS MethodCount
+WHERE MethodCount < 2
+MATCH (c:Character)-[k:Killed]->(c2:Character)
+WHERE k.method = Method
+RETURN Method, c2.name as Victim
+```
+
+6) Daenerys Stormborn of House Targaryen, the First of Her Name, Queen of the Andals and the First Men,
+Protector of the Seven Kingdoms, the Mother of Dragons, the Khaleesi of the Great Grass Sea, the Unburnt,
+the Breaker of Chains or shortened to "Daenerys Targaryen" in our database is by far the biggest killer on
+the show. Let's list all the episodes she killed in as well as characters she killed.
+
+```opencypher
+MATCH (c:Character {name: 'Daenerys Targaryen'})-[k:Killed]->(c2:Character)
+MATCH (c)-[:KillerIn]->(d:Death)<-[:VictimIn]-(c2)
+MATCH (d)-[:InEpisode]-(e:Episode)
+return distinct(c2.name) as Victim, COUNT(d) as Kills, e.name
+ORDER BY Kills DESC
+```
+
+7) Remember that shocking last episode of the fifth season when they killed Jon Snow and we totally thought
+he was gonna stay dead? Well, let's list all the characters that would survive if he actually stayed dead.
+
+```opencypher
+MATCH (c:Character {name: 'Jon Snow'})-[k:Killed]->(c2:Character)
+MATCH (c)-[:VictimIn]->(d2:Death)
+MATCH (c)-[:KillerIn]->(d:Death)<-[:VictimIn]-(c2)
+WHERE d.order>d2.order
+return distinct(c2.name) as Victim, COUNT(d) as Kills
+ORDER BY Kills DESC
+```
+
+8) Houses or allegiances are one of the main aspects of Westeros. We know some houses killed more characters
+than others, but that doesn't matter in the end, what matters is efficiency. Let's list the allegiances with
+the best Kill/Death Ratios. Here we came across one more problem, if an allegiance had more deaths than kills,
+the KD would be 0. This can easily be fixed with the `toFloat()` function.

```

Comment:
When introducing new words like `KD` make sure to have intro to the reader about it. That means adding it to previous sentence `Kill/Death Rations, or short KD ratios`. Also you are missing `ratio`, `KD ratio would be 0`. As KD is not used so much except in one additional place, I would keep full working in second place.